### PR TITLE
Remove learn more link in integration test pages

### PR DESCRIPTION
**Pull request by karthikjeeyar**
## Fixes 

https://issues.redhat.com/browse/HACBS-1169

## Description

Remove learn more link and alert as per the latest UX design doc in the integration test pages.

## Type of change

- [x] Bugfix

## Screen shots / Gifs for design review 

**Add Integration in wizard:**

image

**Add Integration Page:**
image

**Integration List page:**

Screenshot 2022-11-22 at 4 31 01 PM

## How to test or reproduce?

Visit Integration creation/list pages.

### CODE REVIEW

**Comment by MariaLeonova** [comment]
Looks good!

File: src/hacbs/components/IntegrationTestsListView/IntegrationTestsListView.tsx [hunk 1]
```diff
@@ -157,24 +140,6 @@ const IntegrationTestsListView: React.FC<IntegrationTestsListViewProps> = ({ app
...
-                <FlexItem className="pf-u-font-size-sm">
-                  Learn more about setting up an integration test
-                </FlexItem>
-                <FlexItem>
-                  <ExternalLinkAltIcon />
-                </FlexItem>
-              </Flex>
-            }
-          />
-        </FlexItem>
```
Comment (christianvogt) [inline src/hacbs/components/IntegrationTestsListView/IntegrationTestsListView.tsx:177]: nit: removing this flex item leaves the parent flex layout with only a single child. Seems as if we could just remove the flex layout altogether now.

**Pull request merged**